wagtailnhsukfrontend/blocks.py

```diff
@@ -43,3 +43,11 @@ class WarningCalloutBlock(RichTextBlock):
 
     class Meta:
         template = 'wagtailnhsukfrontend/warning_callout.html'
+
+class InsetTextBlock(StructBlock):
+
+    body = RichTextBlock(features=['h1', 'h2', 'h3', 'bold', 'italic', 'link'])
```

Comment:
Looking at the service manual, it is never mentioned. From the HTML template it seems like you should only use <p>, so I will put a change it to remove the headers. As for the bold and italic, it is unclear if you should be able to use them or not. It may be worth discussing this point. As for the links, they are used in the example of inset text, so I think they should be able to be used.

Reply:
I will have a talk with david/ben about this later